src/app/Fake.Azure.Kudu/Kudu.fs

```diff
@@ -55,8 +55,8 @@ let kuduSync() =
     let result =
         Process.execWithResult(fun psi ->
             { psi with
-                FileName = Path.Combine(kuduPath.FullName, "kudusync.cmd")
-                Arguments = sprintf """-v 50 -f "%s" -t "%s" -n "%s" -p "%s" -i ".git;.hg;.deployment;deploy.cmd""" deploymentTemp deploymentTarget nextManifestPath previousManifestPath })
+                FileName = Path.Combine(kuduPath, "kudusync.cmd")
+                Arguments = sprintf """-v 50 -f "%s" -t "%s" -n "%s" -p "%s" -i ".git;.hg;.deployment;deploy.cmd" """ deploymentTemp deploymentTarget nextManifestPath previousManifestPath })
```

Comment:
Another way would be to build an array and use `Args.toWindowsCommandLine` to properly handle all the escaping ugliness of windows

Reply:
Done. Works great: